src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -278,138 +275,137 @@ public final void apply(Project project) {
+                    throw new GradleException(String.format(
+                            "Root lock file '%s' doesn't exist, please run "
+                                    + "`./gradlew --write-locks` to initialise locks",
+                            rootLockfile));
                 }
 
                 configureAllProjectsUsingConstraints(project, rootLockfile, lockedConfigurations);
             });
         }
 
-        TaskProvider verifyLocks = project.getTasks().register("verifyLocks", VerifyLocksTask.class, task -> {
-            task.getCurrentLockState()
-                    .set(project.provider(() -> LockStates.toLockState(fullLockStateSupplier.get())));
-            task.getPersistedLockState()
-                    .set(project.provider(() -> new ConflictSafeLockFile(rootLockfile).readLocks()));
-        });
+        TaskProvider verifyLocks = project.getTasks()
+                .register("verifyLocks", VerifyLocksTask.class, task -> {
+                    task.getCurrentLockState()
+                            .set(project.provider(() -> LockStates.toLockState(fullLockStateSupplier.get())));
+                    task.getPersistedLockState()
+                            .set(project.provider(() -> new ConflictSafeLockFile(rootLockfile).readLocks()));
+                });
         project.getTasks().named(LifecycleBasePlugin.CHECK_TASK_NAME).configure(check -> check.dependsOn(verifyLocks));
 
-        project.getTasks().register("why", WhyDependencyTask.class, t -> {
-            t.lockfile(rootLockfile);
-            t.fullLockState(project.provider(fullLockStateSupplier::get));
-        });
+        project.getTasks()
+                .register("why", WhyDependencyTask.class, t -> {
+                    t.lockfile(rootLockfile);
+                    t.fullLockState(project.provider(fullLockStateSupplier::get));
+                });
     }
 
     static boolean isIgnoreLockFile(Project project) {
         return project.hasProperty("ignoreLockFile");
     }
 
     private static Map<Project, LockedConfigurations> wireUpLockedConfigurationsByProject(Project rootProject) {
-        return rootProject.getAllprojects().stream().collect(Collectors.toMap(Functions.identity(), subproject -> {
-            VersionsLockExtension ext = subproject.getExtensions().getByType(VersionsLockExtension.class);
-            LockedConfigurations lockedConfigurations = computeConfigurationsToLock(subproject, ext);
-            addConfigurationDependencies(subproject,
-                    subproject.getConfigurations().getByName(CONSISTENT_VERSIONS_PRODUCTION),
-                    lockedConfigurations.productionConfigurations());
-            addConfigurationDependencies(subproject,
-                    subproject.getConfigurations().getByName(CONSISTENT_VERSIONS_TEST),
-                    lockedConfigurations.testConfigurations());
-            return lockedConfigurations;
-        }));
+        return rootProject.getAllprojects().stream()
```

Comment:
does it have special-casing to put `.stream()` at the end of a contiguous line???

Reply:
Yes, it first considers `rootProject.getAllprojects()` a prefix (since it's a method on a field/variable), and then it further considers `rootProject.getAllprojects().stream()` to be a prefix because of the special casing you mentioned.

Therefore if this were to break, it would try to make it look like
```java
rootProject.getAllprojects()
        .stream()
```

before falling back to most splits if nothing else fits

```java
rootProject
        .getAllprojects()
        .stream()
```